.github/workflows/security.yaml

```diff
@@ -3,6 +3,8 @@ name: Security
 on:
   push:
   pull_request:
+  schedule:
+    - cron:  '0 0 * * *'
```

Comment:
**suggestion:** Do we really need to run this workflow every day? To encourage digital sobriety, we could run it once a month, perhaps:

```suggestion
    - cron:  '0 0 1 * *'
```

Reply:
Once a week ?